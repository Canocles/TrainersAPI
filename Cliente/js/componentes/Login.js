import React from 'react'

class Login extends React.Component {
    render() {
        return (
            <div className="container" style={{ padding: 10 }}>
                <div className="jumbotron">
                    <h1>Login</h1>
                </div> 
            </div>
        )    
    }
}

export default Login