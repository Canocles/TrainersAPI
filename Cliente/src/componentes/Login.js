import React from 'react'

class Login extends React.Component {
    render() {
        return (
            <div className="row">
                <div className="col-md-4 col-md-offset-4">
                    <form>
                        <h1>Entra en nuestra plataforma</h1>
                        <div className="form-group">
                            <label className="control-label">Email</label>
                            <input ref={campo => this.email = campo} type="email" name="email" placeholder="Introduzca su email" className="form-control" required/>
                        </div>
                        <div className="form-group">
                            <label className="control-label">Password</label>
                            <input ref={campo => this.password = campo} type="password" name="password" placeholder="Introduzca su contraseÃ±a" className="form-control" required/>
                        </div>
                        <a className="btn btn-primary btn-lg" onClick={() => { this.props.handleLogin(this.email.value, this.password.value)}}>
                            Entrar
                        </a>
                    </form>
                </div>
            </div>
        )    
    }
}

export default Login