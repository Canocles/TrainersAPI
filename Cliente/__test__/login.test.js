import React from 'react';
import renderer from 'react-test-renderer'
import Login from '../src/componentes/Login';

describe('Login (snpashot)', () => {
  it('El componente de Login se renderiza correctamente', () => {
    const component = renderer.create(<Login />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
