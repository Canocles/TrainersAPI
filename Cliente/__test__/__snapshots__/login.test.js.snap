// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login (snpashot) El componente de Login se renderiza correctamente 1`] = `
<div
  className="row"
>
  <div
    className="col-md-4 col-md-offset-4"
  >
    <form>
      <h1>
        Entra en nuestra plataforma
      </h1>
      <div
        className="form-group"
      >
        <label
          className="control-label"
        >
          Email
        </label>
        <input
          className="form-control"
          name="email"
          placeholder="Introduzca su email"
          required={true}
          type="email"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="control-label"
        >
          Password
        </label>
        <input
          className="form-control"
          name="password"
          placeholder="Introduzca su contraseÃ±a"
          required={true}
          type="password"
        />
      </div>
      <a
        className="btn btn-primary btn-lg"
        onClick={[Function]}
      >
        Entrar
      </a>
    </form>
  </div>
</div>
`;
